saleor/payment/gateways/authorize_net/tests/conftest.py

```diff
@@ -0,0 +1,72 @@
+from unittest import mock
+
+import pytest
+
+from .....plugins.manager import get_plugins_manager
+from ....interface import GatewayConfig
+from ..plugin import AuthorizeNetGatewayPlugin
+
+
+@pytest.fixture
+def authorize_net_gateway_config():
+    return GatewayConfig(
+        gateway_name="authorize_net",
+        auto_capture=True,
+        supported_currencies="USD",
+        connection_params={
+            "api_login_id": "public",
+            "transaction_key": "secret",
+            "client_key": "public",
+            "use_sandbox": True,
+        },
+    )
+
+
+@pytest.fixture()
+def authorize_net_payment(payment_dummy):
+    return payment_dummy
```

Comment:
I would recommend changing at least the gateway name to authorize net (now it points to dummy) which can cause a lot of trouble in the future development